### PR TITLE
[1LP][WIPTEST]V2v ui tests providers refactoring

### PR DESCRIPTION
**Pull request by kedark3**
Purpose or Intent
=================

__Fixing__ UI test cases  to use NamedTuples for v2v_providers.

{{pytest: cfme/tests/v2v/test_v2v_migrations_ui.py --use-provider rhv42 --use-provider vsphere67-nested --provider-limit 2 -vvvv --long-running}}

### CODE REVIEW

**Comment by kedark3** [comment]
to be merged after #7866

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8a1001970008596e19c00d3bad52a4c9d4c5fd6c:

`cfme/tests/v2v/test_v2v_migrations.py`:
- :red_circle: Line 64:5: **F841** *local variable 'src_vm_ip_addr' is assigned to but never used*

`cfme/tests/v2v/test_v2v_migrations_ui.py`:
- :red_circle: Line 13:1: **F811** *redefinition of unused 'MigrationPlanRequestDetailsView' from line 12*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/v2v/test_v2v_migrations_ui.py [hunk 1]
```diff
@@ -9,7 +9,9 @@
 from cfme.markers.env_markers.provider import ONE_PER_VERSION
 from cfme.utils.appliance.implementations.ui import navigate_to
 from cfme.tests.services.test_service_rbac import new_user, new_group, new_role
-from cfme.v2v.migrations import MigrationPlanRequestDetailsView
+from cfme.v2v.migrations import (
+    MigrationPlanRequestDetailsView, MigrationDashboardView, AddMigrationPlanView
+)
 
 pytestmark = [
     pytest.mark.ignore_stream('5.8'),
```
Comment (sshveta) [inline cfme/tests/v2v/test_v2v_migrations_ui.py]: This can be removed now

File: cfme/tests/v2v/test_v2v_migrations_ui.py [hunk 2]
```diff
@@ -217,31 +219,35 @@ def test_v2v_ui_set2(appliance, v2v_providers, form_data_single_datastore, soft_
...
-    view = navigate_to(migration_plan_collection, 'All')
+    view = appliance.browser.create_view(MigrationDashboardView)
     soft_assert(plan_name in view.migration_plans_not_started_list.read())
-    soft_assert(view.infra_mapping_list.get_associated_plans(mapping.name) == plan_name)
     soft_assert(view.migration_plans_not_started_list.get_plan_description(plan_name) ==
      plan_description)
     # Test Associated Plans count  correctly Displayed in Map List view
     soft_assert(str(len(vm_selected)) in view.migration_plans_not_started_list.
         get_vm_count_in_plan(plan_name))
+    view.browser.refresh()
```
Comment (sshveta) [inline cfme/tests/v2v/test_v2v_migrations_ui.py]: Question : Why is this required ?

Comment (kedark3) [reply cfme/tests/v2v/test_v2v_migrations_ui.py]: This is required because the Associated plans do not show up in infra-mappings-list before refresh.

File: cfme/tests/v2v/test_v2v_migrations_ui.py (hunk 1 above)
Comment (kedark3) [reply cfme/tests/v2v/test_v2v_migrations_ui.py]: Yep.
